.buildkite/build-ios.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -eu
+
+echo '--- :node: Setup Node depenendencies'
+echo '--- :node: 1. Install nvm'
+brew install nvm
+
+echo '--- :node: 2. Load nvm in the current shell'
+export NVM_DIR="$HOME/.nvm"
+mkdir -p "$NVM_DIR"
+[ -s "$HOMEBREW_PREFIX/opt/nvm/nvm.sh" ] && \. "$HOMEBREW_PREFIX/opt/nvm/nvm.sh" --install
+
+echo '--- :node: 3. Install node version from .nvmrc'
+nvm install "$(cat .nvmrc)" && nvm use
+
+echo '--- :node: 4. nmp ci'
+npm ci
```

Comment:
I tried to use the new nvm plugin, but run into a setup issue https://github.com/Automattic/nvm-buildkite-plugin/issues/6 (although it's possible I just misunderstood the plugin target platform)